openqaoa/problems/converters.py

```diff
@@ -19,17 +19,29 @@
 
 
 class FromDocplex2IsingModel:
-    def __init__(self, model, multipliers: Union[float, list] = None):
+    def __init__(self, model, multipliers: Union[float, list] = None, unbalanced_const: bool = False,
+                 strength_ine:list = [0.1, 0.5]):
 
         """
         Creates an instance to translate Docplex models to its Ising Model representation
 
         Parameters
         ----------
         model : Docplex model
-            It is an object that has the mathematical expressions (Cost function,
-            Inequality and Equality constraints) of an optimization problem in
-            binary representation.
+            It is an object that has the mathematical expressions (cost function and
+            inequality and Equality constraints) of an optimization problem in
+            a binary representation.
+        multipliters: [float, integer, list]
+            The strength of the penalties of the cost function
+        heuristic: bool
+            If the method for the inequality constraints is used. This method 
+            implement a novel approach that do not required slack variables.
+        strength_ine: List[float, float]
```

Comment:
perhaps we can name this input `strength_ineq`? Took me a minute to figure out what `strength_ine` meant haha

Reply:
Thank you @vishal-ph, I updated the file.